docs/auth/social.mdx

```diff
@@ -194,6 +194,13 @@ Future<UserCredential> signInWithFacebook() async {
 </TabItem>
 </Tabs>
 
+:::caution
+Firebase will not set the `User.emailVerified` property
+to `true` if your user logs in with Facebook. The user's Facebook provider will be overridden should your user login using a provider that verifies email(e.g. Google).
```

Comment:
```suggestion
to `true` if your user logs in with Facebook. Should your user login using a provider that verifies email (e.g. Google sign-in) then this will be set to true.
```